server/runtime/src/main/java/org/infinispan/server/configuration/security/DistributedRealmConfigurationBuilder.java

```diff
@@ -0,0 +1,65 @@
+package org.infinispan.server.configuration.security;
+
+import static org.infinispan.server.configuration.security.DistributedRealmConfiguration.NAME;
+import static org.infinispan.server.configuration.security.DistributedRealmConfiguration.REALMS;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.infinispan.commons.configuration.Builder;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.wildfly.security.auth.realm.DistributedSecurityRealm;
+import org.wildfly.security.auth.server.SecurityRealm;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 13.0
+ **/
+public class DistributedRealmConfigurationBuilder implements Builder<DistributedRealmConfiguration> {
```

Comment:
Is it okay that this doesn't extend the child builder? This will prevent chaining other method invocations from it.

Reply:
Yeah, the server configuration API is not really for public consumption